chat-api/v2/openapi.yaml

```diff
@@ -1069,25 +1069,83 @@ components:
         replyBack: false
     Media:
       type: object
-      description: Media to be send
+      oneOf:
+        - $ref: '#/components/schemas/AudioMedia'
+        - $ref: '#/components/schemas/ImageMedia'
+        - $ref: '#/components/schemas/DocumentMedia'
+        - $ref: '#/components/schemas/VideoMedia'
+    AudioMedia:
+      type: object
+      description: Audio to be send
       properties:
         type:
           type: string
           description: What kind of media should be send
```

Comment:
a little bit missleading since we are talking about only audio

Reply:
fixed